content/docs/lists-and-keys.md

```diff
@@ -42,15 +42,15 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/GjPyQr?editors=0011)
+[**Chạy thử trên CodePen**](https://codepen.io/gaearon/pen/GjPyQr?editors=0011)
```

Comment:
It shoud be "Thử trên CodePen**" vì chị thấy mấy phần dịch trước mọi người đang dùng từ này rồi.